lib/file-helper.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const fs = require('fs')
+const configPaths = require('../config/paths.json')
+
+// Read components from various sources
+exports.SrcComponentList = fs.readdirSync(configPaths.src + 'components/')
+exports.DistComponentList = fs.readdirSync(configPaths.dist + 'components/')
+exports.PackagesComponentList = fs.readdirSync(configPaths.packages)
+
+const distFilesForComponent = componentName => {
```

Comment:
I wonder if these functions would benefit from doc blocks to explain what they're doing and how they're used?